src/components/SocialListItem.tsx

```diff
@@ -1,42 +1,64 @@
 // Libraries
-import React from "react"
-import styled from "@emotion/styled"
+import React, { useMemo } from "react"
+import { Flex, Box, chakra, Icon } from "@chakra-ui/react"
+import {
+  FaTwitter,
+  FaYoutube,
+  FaDiscord,
+  FaRedditAlien,
+  FaStackExchange,
+  FaGlobe,
+} from "react-icons/fa"
 
 // Components
-import Icon from "./Icon"
+import { socialColors } from "./Icon"
```

Comment:
We are deprecating the usage of our custom `Icon` component. I see that `socialColors` is only being used in this file so lets move those colors in here and avoid the import.